contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -83,9 +83,9 @@ contract UniswapAnchoredView is UniswapConfig {
         reporter = reporter_;
         anchorPeriod = anchorPeriod_;
 
-        require(anchorToleranceMantissa_ < 100e16, "anchor tolerance is too high");
-        upperBoundAnchorRatio = 100e16 + anchorToleranceMantissa_;
-        lowerBoundAnchorRatio = 100e16 - anchorToleranceMantissa_;
+        // Allow the tolerance to be whatever the deployer chooses, but prevent under/overflow (and prices from being 0)
+        upperBoundAnchorRatio = anchorToleranceMantissa_ > uint(-1) - 100e16 ? uint(-1) : 100e16 + anchorToleranceMantissa_;
```

Comment:
Hm, can this logic be made more obvious? Maybe just use `add(anchorToleranceMantissa_, 100e16)`?

Reply:
But if it overflows you need to also tell it what to cap to (and name that fn), so seems like inlining is more clear imo 🤷‍♂️